OSDOCS-18372: DITA abstracts — KMM (#117165)
openshift/openshift-docs: modules/kmm-uninstalling-kmm.adoc

[id="kmm-uninstalling-kmmo_{context}"]
= Uninstalling the Kernel Module Management Operator

[role="_abstract"]

> You can uninstall the Kernel Module Management Operator from {product-title} by using the procedure that matches how you originally installed it.

Reviewer: ```suggestion
You can uninstall the Kernel Module Management (KMM) Operator from {product-title} by using the procedure that matches how you originally installed it.
```

> You can uninstall the Kernel Module Management (KMM) Operator from {product-title} by using the procedure that matches how you originally installed it.

Reviewer 2: Considering GEO, I'm wondering if it might be worthwhile to mention the installation options here. It could make the intro longer than 300 character, though. But it looks like you could do something like...

```suggestion
You can uninstall the Kernel Module Management (KMM) Operator from {product-title} by using the CLI or uninstalling the Operator. 
```

These may not be the only 2 options. 

Only mentioning specificity because I'm considering how a customer might ask a chatbot "How to remove the KMM if I installed it from the CLI."

Author: agree
